docker-compose.yml

```diff
@@ -39,27 +39,32 @@ x-search-api-env: &search-api-env
 
 services:
   nginx-proxy:
-    image: jwilder/nginx-proxy:latest
+    image: jwilder/nginx-proxy:alpine
```

Comment:
We're not planning on using alpine in production btw, but here in the e2e tests it saves a bit of bulk. Also some of these third-party images don't have a debian-slim or similar flavour, so alpine is the only slimline option.